test/cadet/accounts/accounts_test.exs

```diff
@@ -21,15 +21,27 @@ defmodule Cadet.AccountsTest do
   use Cadet.DataCase
   use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
 
-  alias Cadet.{Accounts, Repo}
-  alias Cadet.Accounts.Query
+  alias Cadet.{Accounts, Accounts.Query, Repo}
 
   @token if System.get_env("TOKEN"), do: System.get_env("TOKEN"), else: "token"
 
   setup_all do
     HTTPoison.start()
   end
 
+  test "get or create exisiting user" do
```

Comment:
Put this in a `describe` block since they are describing behaviour of one function.
Then do the test `existing user` or `create new user`